components/alert/test/alert.visual-diff.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<link rel="stylesheet" href="../../../test/styles.css" type="text/css">
+	<script src="/node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
+	<script type="module">
+		import '../../typography/typography.js';
+		import '../alert.js';
+	</script>
+	<title>d2l-alert</title>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<meta http-equiv="X-UA-Compatible" content="ie=edge">
+	<meta charset="UTF-8">
+</head>
+<body class="d2l-typography">
+	<div class="visual-diff">
+		<d2l-alert id="default" type="default">A default message.</d2l-alert>
+	</div>
+	<div class="visual-diff">
+		<d2l-alert id="success" type="success">A success message.</d2l-alert>
+	</div>
+	<div class="visual-diff">
+		<d2l-alert id="critical" type="critical">A critical message.</d2l-alert>
+	</div>
+	<div class="visual-diff">
+		<d2l-alert id="warning" type="warning">A warning message.</d2l-alert>
+	</div>
+	<div class="visual-diff">
+		<d2l-alert id="close" type="error" has-close-button>A message.</d2l-alert>
+	</div>
+	<div class="visual-diff">
+		<d2l-alert id="button" type="call-to-action" button-text="Do it!">A message.</d2l-alert>
+	</div>
```

Comment:
Thanks for including tests for "error" and "call-to-action", which are only supported for backwards-compatibility.